app/models/katello/content_view.rb

```diff
@@ -76,6 +74,7 @@ def <<(*_args)
 
     scoped_search :on => :name, :complete_value => true
     scoped_search :on => :organization_id, :complete_value => true
+    scoped_search :on => :composite, :complete_value => true
```

Comment:
Would it make sense to add some unit tests for scoped-search working on the content views, similarly as done in https://github.com/theforeman/foreman-tasks/blob/master/test/unit/task_test.rb#L16